packages/synapse-react-client/src/components/DownloadCart/DownloadListTable.tsx

```diff
@@ -159,250 +401,50 @@ export default function DownloadListTable(props: DownloadListTableProps) {
       return 'Only Ineligible'
     }
   }
-  const removeItem = async (
-    item: DownloadListItem,
-    fileName: string,
-    title: string,
-  ) => {
-    try {
-      await SynapseClient.removeItemFromDownloadListV2(item, accessToken)
-      displayToast(`${fileName} has been removed from your list.`, 'success', {
-        title,
-      })
-      // refetching the statistics will update the download list, but refresh the list immediately since this will take time
-      refetch()
-      refetchStatistics()
-    } catch (err) {
-      console.error(err)
-    }
-  }
 
   const availableFiltersArray: AvailableFilter[] = [
     undefined,
     'eligibleForPackaging',
     'ineligibleForPackaging',
   ]
   return (
-    <div className="bootstrap-4-backport">
+    <div>
       <BlockingLoader show={copyingAllSynapseIDs} />
       <div className="filterFilesContainer">
         <span className="filterFilesByText">Filter Files By</span>
-        <Dropdown>
-          <Dropdown.Toggle variant="gray-primary-500" id="dropdown-basic">
-            {getFilterDisplayText(filter)}
-          </Dropdown.Toggle>
-          <Dropdown.Menu role="menu">
-            {availableFiltersArray.map(availableFilter => (
-              <Dropdown.Item
-                role="menuitem"
-                key={`${getFilterDisplayText(availableFilter)}-filter-option`}
+        <Select native fullWidth value={getFilterDisplayText(filter)}>
+          {availableFiltersArray.map(availableFilter => (
+            <option
+              key={`${getFilterDisplayText(availableFilter)}-filter-option`}
+              onClick={() => {
+                setFilter(availableFilter)
+              }}
+            >
+              {getFilterDisplayText(availableFilter)}
+            </option>
+          ))}
+        </Select>
+      </div>
+      {allRows.length > 0 && (
+        <div className="DownloadListTableV2">
+          {/* TODO: This table can be very large, so it should be refactored to use row virtualization or discrete pagination */}
+          <StyledTanStackTable table={table} fullWidth={false} />
```

Comment:
Use common TanStack table UI. Left a note for some work that we should do, but is out of scope.